tests/dotnet/NativeFileReferencesApp/Makefile

```diff
@@ -0,0 +1,31 @@
+TOP=../../..
+
+include $(TOP)/Make.config
+
+# This is a temporary variable to enable the .NET workload resolver, because it's opt-in for now.
+# Ref: https://github.com/dotnet/sdk/issues/13849
+export MSBuildEnableWorkloadResolver=true
+
+prepare:
+	cd .. && $(MAKE) global.json NuGet.config
+	rm -Rf */bin */obj
+
+all-ios: prepare
+	$(DOTNET6) build iOS/*.csproj /bl
+
+all-mac: prepare
+	$(DOTNET6) build macOS/*.csproj /bl
+
+run-mac:
+	/Users/rolf/work/maccore/whatever/xamarin-macios/tests/dotnet/NativeReferencesApp/macOS/bin/Debug/net6.0-macos/osx-x64/NativeReferences.app/Contents/MacOS/NativeReferences
```

Comment:
should be relative to `$(TOP)`
your's too far away, well beyond `MAX_PATH` ;-)

Reply:
> your's too far away, well beyond `MAX_PATH` ;-)

🤦